### ISSUE TITLE
Menu will break requiring keys to be held

### ISSUE THREAD

**Issue by Arcanemagus** [steelbrain/intentions#26]
If you press the keys bound to `intentions:show`, then _while the menu is open_ press them again, the menu will disappear... and will require the keys to be held down to show (breaking the ability to use any actions).

**Comment by Arcanemagus** [comment]
Looks like this is partially fixed, the above behavior works correctly now (pressing again hides it, pressing a third time shows it permanently). However if you:
1. Open the menu with the keybind
2. Click on the dialog with the mouse (which doesn't activate the action, separate bug?)

Then the dialog will go away, and attempting to run the activation key binds again will result in the original behavior of them requiring to be held down.

**Comment by steelbrain** [comment]
Fix is in master but not gonna rush the release this time (also waiting on new specs covering this)

**Comment by Arcanemagus** [comment]
Looks like that fixed it for me, should I file another issue about clicking the option just closing the menu (instead of performing the action)?

**Comment by Arcanemagus** [comment]
Filed as a separate issue, closing this as it's fixed on `master`.

**Comment by steelbrain** [comment]
Keep this open till specs are in

**Comment by steelbrain** [comment]
Just caught a bug in highlights writing specs, so ha! :D Specs are nice
